### ISSUE TITLE
use compiled releases similar to bosh-deployment

### ISSUE THREAD

**Issue by cppforlife** [concourse/concourse-bosh-deployment#8]
nice quick way to compile releases inside concourse: https://github.com/cloudfoundry/bosh-deployment/tree/master/ci/compiled-releases

**Comment by jtarchie** [comment]
To follow along concourse/concourse#1380

**Comment by vito** [comment]
Is there any way to publish compiled releases to bosh.io? I'm trying to have that site be the centerpiece of our release, and have this repo point everything to it. It'd be nice to avoid an s3 bucket and have the compiled releases come from the source of truth.

**Comment by srbry** [comment]
@vito has any progress been made on this? It would be really useful, especially when doing the `lite` deployment. cf-deployment has an ops-file that plugs it in for example.

I would agree that using bosh.io for compiled releases would be nice but s3 seems like a good fall back given how long this issue has already been open?

**Comment by vito** [comment]
@srbry This issue is still open because it's not a priority, not because we're waiting on bosh.io to do it for us.

We could make compiled releases and put them in s3 but it's a nonzero amount of effort to set up and maintain forever. The pain here is minimal enough and we've simply got plenty of other stuff to work on that more people care about. I can't justify shifting our small team's focus away from things like spaces/new login flow/UX redesign for an optimization that only affects BOSH deploys.

If there's some way that a PR could be submitted for this (maybe a pipeline?) then we could pull it in.

**Comment by srbry** [comment]
@vito that makes total sense. It wasn't clear from the prior conversation if this was had an amount of work in progress or not. Thanks for taking the time to reply.

**Comment by taylorsilva** [comment]
Stale, closing.
